lib/common/utils.cpp

```diff
@@ -251,3 +252,24 @@ bool Wasp::BinarySearchRange(
 	i = i0;
 	return(true);
 }
+
+
+bool Wasp::NearlyEqual(float a, float b, float epsilon) {
+	float absA = fabs(a);
+	float absB = fabs(b);
+	float diff = fabs(a-b);
```

Comment:
Ironically, `fabs` is the double abs function. The float abs function is `fabsf`.

Reply:
As of C++11 fabs is overloaded to work on 32bit floats (and other types)

Reply:
Oh, cool.

Reply:
From a good programming habit standpoint, if you want to use C++11 version of `fabs`, you need to specify `std::fabs()`. That's because, on Linux, the code above will in fact invoke the C language version of fabs.

From a practical standpoint, you can argue the wasted cycles dont matter.

P.s. you can run the following code to verify which fabs is invoked.

```C++
#include <iostream>
#include <type_traits>
#include <cmath>

int main()
{
    float f = 3.5f;
        
    auto v1 = fabs( f );
    auto v2 = std::fabs( f );

    std::cout << std::boolalpha;
    std::cout << std::is_same< decltype(v1), double >::value << '\n';
    std::cout << std::is_same< decltype(v2), double >::value << '\n';
}
```

Reply:
Good point. Done.